From: Anand Jain <anand.jain@oracle.com>
Date: Thu, 10 Mar 2016 12:22:15 +0800
Patch-mainline: 4.7
Git-commit: ad8403df054c9b5dc3175a26c6179571b9cafa4e
References: bsc#1012452
Subject: [PATCH] btrfs: pass the right error code to the
 btrfs_std_error

Also drop the newline from the message.

Signed-off-by: Anand Jain <anand.jain@oracle.com>
Signed-off-by: David Sterba <dsterba@suse.com>
---
 fs/btrfs/ioctl.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/fs/btrfs/ioctl.c
+++ b/fs/btrfs/ioctl.c
@@ -4983,8 +4983,8 @@ static long btrfs_ioctl_qgroup_assign(st
 	/* update qgroup status and info */
 	err = btrfs_run_qgroups(trans, root->fs_info);
 	if (err < 0)
-		btrfs_handle_fs_error(root->fs_info, ret,
-			    "failed to update qgroup status and info\n");
+		btrfs_handle_fs_error(root->fs_info, err,
+			    "failed to update qgroup status and info");
 	err = btrfs_end_transaction(trans, root);
 	if (err && !ret)
 		ret = err;
